### PR TITLE
Madaline & Telmoda Crash Fix

### PR DESCRIPTION
**Pull request by Era-Lusiphur**
Resolves this small issue: https://github.com/project-topaz/topaz/issues/1055

Zach EDIT: fixes #1056 

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/zones/Northern_San_dOria/npcs/Telmoda.lua [hunk 1]
```diff
@@ -22,4 +22,4 @@ function onEventUpdate(player, csid, option)
 end
 
 function onEventFinish(player, csid, option)
-end
+end
```
Comment (zach2good) [inline scripts/zones/Northern_San_dOria/npcs/Telmoda.lua]: Please put this newline back 🙏

**Review by zach2good — APPROVED** [review]
Looks good, just fix that newline please 👍

**Comment by zach2good** [comment]
Also, here's a sneaky tip: If you write:
Fixes #1056 
It'll link to the issue and close it when this gets merged (I think)
EDIT (Link): https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords#:~:text=issue%20will%20close.-,Closing%20an%20issue%20in%20the%20same%20repository,merged%20into%20the%20default%20branch.

File: scripts/zones/Northern_San_dOria/npcs/Telmoda.lua (hunk 1 above)
Comment (Era-Lusiphur) [reply scripts/zones/Northern_San_dOria/npcs/Telmoda.lua]: WHOOPS.  :P

**Comment by zach2good** [comment]
There we go :)

**Comment by zach2good** [comment]
I did a search in scripts for `player:setCharVar(player,` and these were the only two scripts that had this leftover syntax (and crash), so it's fixed everywhere with this PR 🙌

**Review by Kreidos — APPROVED** [review]
Yep! 

:bricks: :car:

**Pull request merged**